astro_2.0/features/Comments/components/Comment/Comment.tsx

```diff
@@ -83,14 +94,25 @@ export const Comment: FC<CommentProps> = ({
         )}
         <div className={styles.right}>
           <div className={styles.likes}>
-            {likeAccounts.length > 0 ? likeAccounts.length : null}{' '}
+            {likeAccounts.length}
             <IconButton
+              size="medium"
               disabled={!accountId}
-              icon={isLikedByMe ? 'heartFilled' : 'heart'}
-              className={cn(styles.likeIcon, { [styles.liked]: isLikedByMe })}
+              icon={isLikedByMe ? 'likeFilled' : 'like'}
+              className={styles.likeIcon}
               onClick={() => onLike(id, isLikedByMe)}
             />
           </div>
+          <div className={styles.likes}>
+            {dislikeAccounts.length}
+            <IconButton
+              size="medium"
+              disabled={!accountId}
+              icon={isDislikedByMe ? 'likeFilled' : 'like'}
+              className={cn(styles.likeIcon, styles.dislikeIcon)}
+              onClick={() => onDislike(id, isDislikedByMe)}
+            />
+          </div>
```

Comment:
What do you think about bringing this block to a separate component (nested inside this directory) and reusing it? Just a thought. Not mandatory.

Reply:
Fixed